ci/travis/install.sh

```diff
@@ -22,5 +22,5 @@ export HOMEBREW_NO_AUTO_UPDATE=1
 
 if [ ${TRAVIS_OS_NAME} == "osx" ]; then
     brew install opencv
-    python -m pip install --user nose numpy cython scipy requests mock nose-timer nose-exclude
+    python -m pip install --user nose numpy cython scipy requests mock nose-timer nose-exclude mxnet-to-coreml
```

Comment:
I don't think we should pip install it. Otherwise, changes in a PR will not be caught. Instead, it should use the package of the source package as pointed out by kellen

Reply:
Hi @marcoabreu, currently this converter is not built in the source together with MXNet. I am not sure the historical reason but I guess maybe it is only used for one particular group of users (iOS devs)? Without this CI, we have no way to even know if we should update the PyPI package. Thanks!

Reply:
Hi @marcoabreu, the purpose of adding this to CI is to make sure the latest change is updated in the mxnet-to-coreml PyPi package. It is not currently built together with MXNet source code.

Reply:
The problem is that a PR to update the core ml converter might introduce a bug because we are not actually validating the package but rather the released package. Since the package is under our control and the purpose is to validate it, we should use the source package as source instead of the released package.

Otherwise, we will have all PRs and pipelines suddenly dial as soon as we publish a new release of the converter. I don't think that's the right flow for a test.

Reply:
@marcoabreu  The purpose of this CI is not to validate the coreml package but to validate that the coreml package on PyPi can work with the latest MXNet release. There had been an issue earlier that breaks customer flow because coreml package was not up-to-date on PyPI. Without this validation in CI, the same problem could happen again in the future. In addition, mxnet-to-coreml converter package is in maintenance mode so I do not foresee a frequent update in this package.

Reply:
Okay, so I'd rather categorize them as integration tests and pull them into a different pipeline. Validating a "third party" package in maintenance mode during PR stage doesn't seem to be the right thing to me to be honest.

One problem here is the fact that we currently don't have a nightly stage for Travis. We might have to look into that and in how far it's possible to provide that.

Reply:
Yeah exactly what I'm thinking.  I see adding this check as a two way door though.  I'd be ok with adding it per PR (as it runs quite quickly) now, and then migrating it when we get Travis setup on nightly.

Reply:
Looks like, as usual, Travis provides a simple method of doing nightlies.   https://docs.travis-ci.com/user/cron-jobs/

Reply:
@KellenSunderland Your understanding is very accurate. Thanks!
@marcoabreu Could you suggest a proper place to put these tests? If we currently don't have a nightly stage, can we merge this test in for now and move them to nightly later on? Thanks for your time!

Reply:
@lebeg can you help out here?

Reply:
@lebeg any suggestion? Thanks

Reply:
I'm fully with @KellenSunderland on this. The current place is not ideal, but it's the best we can get for now. I suggest we create a ticket for moving this to nightlies once they will get set up.

Reply:
@marcoabreu I have created a ticket (https://issues.apache.org/jira/browse/MXNET-1103) to continue improving this package in CI. Could you please merge this PR so that it can at least help to provide coverage for existing customers? Thanks